### PR TITLE
Implemented Wheel class.

### PR DESCRIPTION
**Pull request by FarzanRashid**
This merge request contains wheel class. Also adds following dependencies

- pylint
- mypy
- black

### CODE REVIEW

File: src/roulette.py [hunk 1]
```diff
@@ -49,10 +51,78 @@ def winAmount(self, amount: float) -> float:
...
+
+    def __init__(self) -> None:
+        """
+        Creates a new wheel with 38 empty Bin instances. It will also create a new random number
+        generator instance.
+
+        At the present time, this does not do the full initialization of the Bin instances. We’ll
+        rework this in a future exercise.
+        """
+        self.bins = tuple(Bin() for i in range(38))
```
Comment (FarsanRashid) [inline src/roulette.py:92]: unnecessary `i`.

**Review by FarsanRashid — COMMENTED** [review]
The scope of doc string formatting of `Bin` and `Outcome` class should have been a separate task(refactoring) than Wheel implementation. The reviewing(readability) was difficult due to lots of changes which are coming from doc string changes.

File: src/roulette.py (hunk 1 above)
Comment (FarsanRashid) [inline src/roulette.py:92]: unnecessary `i`

Comment (FarsanRashid) [inline src/roulette.py:92]: try to avoid magic literals. don't need to change for this one as the doc string is explaining what magic literal 38 is about.

Comment (FarzanRashid) [reply src/roulette.py]: Done. Thanks.

Comment (FarzanRashid) [reply src/roulette.py]: Noted. Thanks.

**Pull request merged**